solvers/sgd_torch.py

```diff
@@ -14,14 +14,18 @@ class Solver(TorchSolver):
     parameters = {
         'nesterov, momentum': [(False, 0), (True, 0.9)],
         'lr': [1e-1],
+        'coupled_weight_decay': [0.0, 1e-4],
+        'decoupled_weight_decay': [0.0, 1e-4],
```

Comment:
I would simply put `weight_decay` to avoid duplicating the run.
```suggestion
        'weight_decay': [0.0, 1e-4],
```

Reply:
Actually here I wanted to duplicate the run, in order to have an easier time groupping the plots. Indeed since the 2 are equivalent for SGD, it would make sense to compare each one to their application to Adam, wdyt?